lib/msal-common/src/auth/authority/B2cAuthority.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Authority } from "./Authority";
+import { AuthorityType } from "./AuthorityType";
+import { ClientConfigurationError } from "../../error/ClientConfigurationError";
+import { INetworkModule } from "../../network/INetworkModule";
+
+export const B2CTrustedHostList: string[] = [];
```

Comment:
Could this instead be a static member on the b2cauthority class?

Reply:
Done